showcase/treemap/dynamic-treemap.js

```diff
@@ -19,7 +19,7 @@
 // THE SOFTWARE.
 
 import React from 'react';
-import window from 'global/window';
+import timers from 'timers';
```

Comment:
Didn't knew this one, thanks. Don't it need to be shimmed on the client like `fs`?